pkg/auth/auth_handler.go

```diff
@@ -50,6 +55,12 @@ func NewAuthHandler(cfg Config, ocmCLient *ocm.Client, log logrus.FieldLogger) *
 	return a
 }
 
+func getAdminUsers(cfg Config, log logrus.FieldLogger) []string {
+	admins := strings.ReplaceAll(cfg.AdminUsers, " ", "")
```

Comment:
I think that this is not necessary.
all you need to do is set `AdminUsers` as `[]string` in the config structure and envconfig should do the parsing